andela/titan-dev-ah-frontend
# #163518642  Users should be able Signup/Register to Authors Haven

#### What does this PR do?
Build registration front end feature
#### Description of Task to be completed?
-  Create SignUp components with React 
- Integrate the component with redux for state management
- write unit tests for all elements used by the component 
#### How should this be manually tested?
- clone the repo and check out the following branch ft-user-login-163518644
- run `yarn start` and navigate to /register in your browser
- Submit valid credentials to be registered
- Submit invalid credentials to get errors
- run  `yarn test` to  run the unit tests
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163518642 
#### Screenshots (if appropriate):
Coming Soon
#### Questions:
